services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -218,6 +219,14 @@ private long getHeadSlotValue() {
     return recentChainData.getBestSlot().longValue();
   }
 
+  private long getHeadRootValue() {
+    Optional<Bytes32> maybeBlockRoot = recentChainData.getBestBlockRoot();
+    if (maybeBlockRoot.isPresent()) {
+      return maybeBlockRoot.get().getLong(24, ByteOrder.LITTLE_ENDIAN);
```

Comment:
There are a few roots that get published by metrics (I think), so should this be a static utility method to do the conversion?

Reply:
yep there's a few, I'll look at refactoring the shared code when i get into them :)